modules/nw-http2-haproxy.adoc

```diff
@@ -6,31 +6,37 @@
 [id="nw-http2-haproxy_{context}"]
 = Enabling HTTP/2 Ingress connectivity
 
-You can enable transparent end-to-end HTTP/2 connectivity in HAProxy. It allows application owners to make use of HTTP/2 protocol capabilities, including single connection, header compression, binary streams, and more.
+You can enable transparent end-to-end HTTP/2 connectivity in HAProxy, so that an application owner can access HTTP/2 protocol capabilities, which includes single connection, header compression, binary streams, and other features.
```

Comment:
I'm not sure that it is useful to list capabilities.  If I didn't already know what you meant by "single connection", I would not understand from this statement, and I might even infer something false.  

(HTTP/1.1 supports pipelining, which allows multiple serial requests over a single TCP connection.  HTTP/2 adds multiplexing of multiple concurrent requests over a single TCP connection.  I'm guessing that "single connection" is referring to the latter.)

Reply:
Updated to "such as multiplexing of multiple concurrent requests over a single Transmission Control Protocol (TCP) connection."